draft-irtf-cfrg-hpke.md

```diff
@@ -368,9 +368,8 @@ context. The key schedule inputs are as follows:
   "")
 * `psk` - A pre-shared secret held by both the sender
   and the recipient (optional; default value `zero(Nh)`).
-* `pskID` - An identifier for the PSK (optional; default
-  value `"" = zero(0)`
-* `pkS` - The sender's public key (optional; default
+* `pskID` - An identifier for the PSK (optional; default value `""`)
```

Comment:
```suggestion
* `pskID` - An identifier for the PSK (optional; default value `zero(0)`)
```

Reply:
That works as well. I would change it for `info` as well then, to zero(0)?

Reply:
Well, pskID is an opaque byte string and has a default value in the pseudocode (`zero(0)`), whereas `info` is traditionally considered to be a UTF8-encoded string. Maybe we could clarify that instead?  @bifurcation, penny for your thoughts?

Reply:
Ah you are right, I forgot about the default for pskID. Given that, I like your suggestion to have zero(0) for pskID and "" for info.